tests/contrib/open_telemtry/test_open_telemetry_middleware.py

```diff
@@ -0,0 +1,129 @@
+from time import sleep
+from typing import Any, Tuple, cast
+
+from opentelemetry.metrics import get_meter_provider, set_meter_provider
+from opentelemetry.sdk.metrics._internal import MeterProvider
+from opentelemetry.sdk.metrics._internal.aggregation import (
+    ExplicitBucketHistogramAggregation,
+)
+from opentelemetry.sdk.metrics._internal.export import InMemoryMetricReader
+from opentelemetry.sdk.metrics._internal.instrument import Counter
```

Comment:
```suggestion
from opentelemetry.sdk.metrics import Counter, MeterProvider
from opentelemetry.sdk.metrics.view import (
    ExplicitBucketHistogramAggregation,
)
from opentelemetry.sdk.metrics.export import InMemoryMetricReader
```

Reply:
reason for this is that there is no `__all__` in these files, and mypy is set to require explicit re-export on our end. I will leave this as is for now since this is a testing code. Would you consider adding `__all__ = (...)` to the packages?

Reply:
> Would you consider adding __all__ = (...) to the packages?


They should be there, but yeah, we will look into this.